pkgs/development/python-modules/nocturne/default.nix

```
@@ -10,18 +10,18 @@
   pyvirtualdisplay,
   sfml,
   substituteAll,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "nocturne";
-  version = "unstable-2022-10-15";
-  format = "setuptools";
+  version = "unstable-2024-06-19";
 
```

```suggestion
  pyproject = true;
```
Since you removed `format = "setuptools"`